src/gui/CMakeLists.txt

```diff
@@ -1713,6 +1714,10 @@ target_link_libraries(qgis_gui
   ${QT_VERSION_BASE}::MultimediaWidgets
 )
 
+if (HAVE_ANALYSIS)
```

Comment:
I'm a big -1 to this -- gui should not be linking to analysis code. What's the reasoning here?